app/utils/xpath.py

```
@@ -71,7 +71,7 @@ class MarketValue:
         RANKINGS_POSITIONS = "//span[contains(@class, 'quick-fact__content--large')]//text()"
 
     class Transfers:
-        YOUTH_CLUBS = "//div[@data-viewport='Jugendvereine']//div//text()"
+        YOUTH_CLUBS = "//div[@class='box tm-player-additional-data'][descendant::*[contains(text(), 'Youth')]]//div[@class='content']//text()"
```

Ruff

Line too long (142 > 120)

(E501)